_pages/install/on_openbsd.md

```diff
@@ -26,8 +26,8 @@ cd /usr/ports/lang/crystal
 doas make clean install
 ```
 
-To avoid building dependencies from source (which can take a long time), you can first install them from binary packages:
+To avoid building LLVM from source (which can take a long time), you can first install the binary package:
 
 ```bash
-doas pkg_add llvm libiconv boehm-gc libevent2 pcre libyaml
```

Comment:
why the other dependencies aren't needed?

Reply:
The packages will still be installed as they're dependencies of the port. The recommendation to install the binary package is mostly important for `llvm` which takes very long to build from source. The others are not as critical, so I don't think we need to explicitly mention them. This is the same in the FreeBSD instructions.